UT4MasterServer.Web/vite.config.ts

```diff
@@ -7,7 +7,8 @@ export default defineConfig(({ command, mode }) => {
   const viteConfig = {
     define: {
       __BACKEND_URL: JSON.stringify(viteEnv.VITE_API_URL),
-      __WEB_BASIC_AUTH: JSON.stringify(viteEnv.VITE_BASIC_AUTH)
+      __WEB_BASIC_AUTH: JSON.stringify(viteEnv.VITE_BASIC_AUTH),
+      __RECAPTCHA_SITE_KEY: JSON.stringify(viteEnv.VITE_RECAPTCHA_SITE_KEY)
```

Comment:
Add it to reduce diff for next PR
```suggestion
      __RECAPTCHA_SITE_KEY: JSON.stringify(viteEnv.VITE_RECAPTCHA_SITE_KEY),
```

Reply:
I don't play hanging commas, against my religion